### ISSUE TITLE
Licenses for included libraries

### ISSUE THREAD

**Issue by JayFoxRox** [mhsjlw/mcpilauncher#12]
You are distributing other peoples works but don't include the respective licenses anywhere: https://github.com/mhsjlw/mcpilauncher/tree/master/natives/lib

**Comment by mhsjlw** [comment]
Yeah, I was debating how to handle this. I'm thinking I'll just add a makefile target or script that downloads the same binaries from the Debian repositories. Besides, that isn't the only licensing issue this repository has. This needs to be resolved ASAP, though

**Comment by mhsjlw** [comment]
Going to wipe these off out of the repository and add a script that asks the user to accept all the licenses of the required libraries, then download them off the Debian repositories.
